README.md

```diff
@@ -80,4 +80,25 @@ You will need to deploy the application using Ansible you need to copy the `ansi
 ```bash
 ansible-playbook playbook.yaml -i inventory.yaml
 ```
-> You must setup passwordless SSH Access from the Host to your Control Nodes
\ No newline at end of file
+> You must setup passwordless SSH Access from the Host to your Control Nodes
+
+## Building & Running the App via Docker
+To build the container for local development, please run 
+```bash
+docker build --target dev --tag todo_app:dev -f dockerfiles/Dockerfile.todo_app .
+```
+
+## To run the container for local development, please run
+```bash
+docker run --env-file ./.env -p 5100:80 --mount "type=bind,source=$(pwd)/todo_app,target=/app/todo_app" todo_app:dev
```

Comment:
I think you've got a port mismatch here; with this command I can't see your app running!
```suggestion
docker run --env-file ./.env -p 5100:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/app/todo_app" -it todo_app:dev
```
(Also a mild recommendation to add `-it` which makes it easier to stop a container directly with ctrl+c)